grott/script.sh

```
@@ -256,4 +256,5 @@ if bashio::config.true 'grott_mqtt'; then
     export gmqttpassword="$GROTT_CONFIG_MQTT_PASSWORD"
 fi
 
-python -u grott.py -c "$DATA_PATH/grott.ini"
+cd $DATA_PATH
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)